### ISSUE TITLE
Build solution fails with 'Microsoft.Office.Core' not found

### ISSUE THREAD

**Issue by tim-wovn** [MicrosoftTranslator/DocumentTranslator#90]
I'm trying to build the DocumentTranslator solution but it fails with Microsoft.Office.Core not found.  I think I've installed all the prerequisites for this project, but I am not familiar with .net development.

I searched online with my error message and found some discussion on stack overflow that the Microsoft.Office.Core libraries are provided with Microsoft Office.  I don't have it installed on my machine at the moment and no Microsoft.Office references are available for me to include under "COM".

Is Microsoft Office required to build the library?  If so, does it need to be any particular version and can this be added to the requirements list on the readme?

**Comment by chriswendt1** [comment]
I think I have never tried it on a machine without Office, so I don't know for sure. At runtime Office is required to translate doc, xls, ppt (those are the pre-OpenXML file formats) and for pdf files. Office is used to convert these to OpenXML format, and only then they can be translated. DocumentTranslator doesn't need Office for any other file formats. 
It may very well be that something at build time checks for these. If you do not need to translate those file formats, you could consider removing the whole Office interop logic, or put it behind a compile time switch.

**Comment by tim-wovn** [comment]
That's good to know.  Thank you for the information.  For my case, I kept digging around and was able to build after reading this information:
https://stackoverflow.com/a/47881824/10448969

Instead of referencing the comm 'Microsoft.Office.Core' there is an assembly simply called 'office' that can be installed with the Visual Studio Installer which contains the Core.

After that, I was getting lots of errors like `The name "Interaction" does not exist in the namespace...`.  I had to install Blend from Visual Studio Installer and then link `System.Windows.Interactivity 4.0.0.0`

Now it builds just fine.  I'm not sure if this configuration is very specific to my case or not.  Maybe it will help out other people.


To summarize:

Open the `Visual Studio Installer` and add
+ Office/SharePoint development
+ Blend for Visual Studio SDK for .NET

Open `Add to References` (for the subprojects described in the error)
In `Assemblies` >> `Extensions` add
+ office 15.0.0.0
+ System.Windows.Interactivity 4.0.0.0 (4.5.0.0 will not work)

**Comment by chriswendt1** [comment]
Can you make a PR to master with your changes? I think it is beneficial if Office is not required for building. It is still required for testing.

**Comment by tim-wovn** [comment]
This weekend, I'll try again to build again on my system using a clean repository.  If all goes well, I'll make a PR with the changes.  Thanks!
